### DISCUSSION TITLE
how to open several tabs at once

### DISCUSSION THREAD

**Discussion by LashiCD** [Eugeny/tabby#8529] [Q&A]
Is there a way to open several tabs at once, each in a different directory, and be able specify the shell being used?

I have several git repos, and it would be nice to run a command/script/something and open tabs for each of the repos open and ready to go.

**Comment by lawweiliang** [comment]
Hi  @LashiCD ,

Here is the step to get multiple tab with multiple workspace/path

![image](https://github.com/Eugeny/tabby/assets/34936929/59987404-ab3d-44b7-9970-5a14d165940b)
